components/src/atoms/link/Link.scss

```diff
@@ -10,7 +10,7 @@ $link__transition           : $transition-base !default;
 $link__color--inverted      : $gray-darker !default;
 $link__color-hover--inverted: $blue !default;
```

Comment:
Please assign variables from `&--inverted` styles to global variables as in line above

Reply:
Sorry, I had something else in mind :D In line 35 for `&--secondary` we have these styles:
```
&--secondary {
    color: $gray-lighter;

    &:hover,
    &:focus {
      color: $color-primary;
    }
  }
```
please assign these colors to new variables to have them defined on the top